gutenberg/blocks/ts/src/linkedevents/api.ts

```diff
@@ -101,7 +101,7 @@ export class LinkedEventsApi {
    * @returns result
    */
   private fetchFromLinkedEvents = async (url: string) => {
-    const result = await (await fetch(`${this.apiUrl}/${url}`, {
+    const result = await (await fetch(`${this.apiUrl}${url}`, {
```

Comment:
Issue mentioned in previous comment is still present. Base URL is passed directly from settings into the Gutenberg block so this is probably a configuration issue and can break existing installations.

You should ensure that there is only one slash here instead of just removing one from the code